src/schema/objects/suffixes.yaml

```
@@ -877,3 +877,40 @@ unloc:
   description: |
     MRS acquisitions run without localization.
     This includes signals detected using coil sensitivity only.
+audio:
+  value: audio
+  display_name: Audio stimulus file
+  description: |
+    Any data file used as an audio stimulus. Media files usually have an audio file type (such as wav, mp3, aac, ogg).
+    The JSON file associated with each media file should contain information such as License (RECOMMENDED), Copyright
+    (RECOMMENDED), URL (OPTIONAL), and Description (OPTIONAL) to describe the origin and the nature of the media.
```

I do not think it is advisable to describe all the fields and whether they are required or not -- this duplicates information formalized elsewhere so would be duplication and thus prone to get out of date etc.  

```suggestion
    The JSON file associated with each media file should contain information to describe the origin and the nature of the media.
```

here and similarly below for others

---

Addressed in 8178a3f98241af1ea53ff145d457ab6508949db3